tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -593,6 +743,50 @@ Status FragmentMetadata::store(const EncryptionKey& encryption_key) {
     }
   }
 
+  // Store mins
+  if (version_ >= 11) {
```

Comment:
This seems like a good time to introduce the versioned class or at minimum store function. What is your thoughts on that here?

Reply:
@Shelnutt2 Good point! Added the versioning on the store function. But yes, file is getting quite large and will be much better to have class versioning in the near future!